orm/encoding/ormkv/unique_key.go

```diff
@@ -64,6 +64,13 @@ func NewUniqueKeyCodec(prefix []byte, messageType protoreflect.MessageType, inde
 		}
 	}
 
+	// if there is nothing in the value we have a trivial unique index
```

Comment:
```suggestion
	// if there is nothing in the value we have a trivial index
```

Reply:
It is not a trivial index, it is just not a new unique constraint